freshmaker/lightblue.py

```diff
@@ -767,17 +770,18 @@ def filter_out_images_with_higher_srpm_nvr(self, images, srpm_name_to_nvrs):
                 ret.append(image)
             image_included = False
             for rpm in rpms or []:
-                image_srpm_nvr = kobo.rpmlib.parse_nvr(rpm["srpm_nevra"])
-                for srpm_nvr in srpm_name_to_nvrs.get(rpm.get("srpm_name"), []):
-                    input_srpm_nvr = kobo.rpmlib.parse_nvr(srpm_nvr)
+                # Not sure about which NVR we should parse srpm or common rpm nvr?
+                image_rpm_nvr = kobo.rpmlib.parse_nvr(rpm["nvra"])
```

Comment:
you need to remove the 'arch' part from nvra before `parse_nvr`.

Reply:
I will just replace it with `parse_nvra` method, if it's appropriate.